sklearn/manifold/t_sne.py

```diff
@@ -808,17 +812,27 @@ def _tsne(self, P, degrees_of_freedom, n_samples, random_state,
         # * early exaggeration with momentum 0.5
         # * early exaggeration with momentum 0.8
         # * final optimization with momentum 0.8
+
+        # TODO sklearn 0.20: remove this warning
+        if self.learning_rate == "200 (new default)":
```

Comment:
From @jnothman's comment in https://github.com/scikit-learn/scikit-learn/pull/8397#discussion_r111121573:

> The only problem is that you're doing this in `__init__` when you should be doing it in `fit`.